app/views/links/edit.html.erb

```diff
@@ -0,0 +1,21 @@
+<% breadcrumb :links, @local_authority, @service, @interaction %>
+
+<%= render partial: "shared/local_authority_details", locals: {authority: @local_authority} %>
+
+<div class="page-title">
+  <h2><%= @interaction.label %></h2>
+</div>
+<b>LGSL</b> <%= @service.lgsl_code %>
+<hr>
+<b>LGIL</b> <%= @interaction.lgil_code %>
+<hr>
+
+<%= label_tag 'link', 'URL' %>
+
+<%= form_for @link, url: {action: "update"} do |f| %>
```

Comment:
I don't think this `url: {action: "update"}` is doing anything as when `@link` is unsaved we end up in the `create` action, and when it is saved we end up in the `update` action.  This is the default behaviour of `form_for` so our `url` param isn't doing anything (and acts to confuse the reader as they'll think we always end up in the `update` action which isn't the case).

That said, if our intention is to always go through the `update` action regardless of it being a saved or unsaved (not a terrible idea if we can do something with @davidbasalla's idea of using some kind of `first_or_intialize` in / around our `Link.get_link` call) then this is the right sort of thing to do, but it's not working - maybe worth exploring this a little bit more.